upgrading/manual_upgrades.adoc

```diff
@@ -507,80 +507,41 @@ updated.
+During a cluster upgrade, the default roles
 xref:../architecture/additional_concepts/authorization.adoc#roles[default
-cluster roles] are automatically updated. To check if all defaults are set as
-recommended for your environment, run:
-
-----
-# oc adm policy reconcile-cluster-roles
-----
+cluster roles] are automatically updated. 
 
 [WARNING]
 ====
-If you have customized default cluster roles and want to ensure a role reconciliation
-does not modify those customized roles, annotate them with `openshift.io/reconcile-protect`
-set to `true` when using the old Openshift policy format. When using the new RBAC
-roles, use `rbac.authorization.kubernetes.io/autoupdate` set to `false` instead.
-In doing so, you are responsible for manually updating those roles with any new
-or required permissions during upgrades.
-====
-
-This command outputs a list of roles that are out of date and their new proposed
-values. For example:
-
-----
-# oc adm policy reconcile-cluster-roles
-apiVersion: v1
-items:
-- apiVersion: v1
-  kind: ClusterRole
-  metadata:
-    creationTimestamp: null
-    name: admin
-  rules:
-  - attributeRestrictions: null
-    resources:
-    - builds/custom
-...
-----
+If you customized default cluster roles and want to ensure a role reconciliation
+does not modify those customized roles, set the
+`rbac.authorization.kubernetes.io/autoupdate` parameter in the 
+to `false`.
 
-[NOTE]
-====
-Your output will vary based on the OpenShift version and any local
-customizations you have made. Review the proposed policy carefully.
+You must manually update the roles with this setting to include any new
+or required permissions after upgrading.
 ====
 
-You can either modify this output to re-apply any local policy changes you have
-made, or you can automatically apply the new policy using the following process:
 
-. Reconcile the cluster roles:
-+
-----
-# oc adm policy reconcile-cluster-roles \
-    --additive-only=true \
-    --confirm
-----
+To check if all defaults are set as recommended for your environment, run the
+following commands:
 
-. Reconcile the cluster role bindings:
+. Generate a default bootstrap policy template file:
 +
 ----
-# oc adm policy reconcile-cluster-role-bindings \
-    --exclude-groups=system:authenticated \
-    --exclude-groups=system:authenticated:oauth \
-    --exclude-groups=system:unauthenticated \
-    --exclude-users=system:anonymous \
-    --additive-only=true \
-    --confirm
+$ oc adm create-bootstrap-policy-file --filename=policy.json
 ----
 +
-Also run:
+[NOTE]
+====
+The contents of the template file vary based on the OpenShift version and any local
+customizations you made. Review the proposed policy carefully.
+====
+
+. Use the policy file to automatically reconcile the roles and role bindings:
 +
 ----
-# oc adm policy reconcile-cluster-role-bindings \
-    system:build-strategy-jenkinspipeline \
-    --confirm \
-    -o name
+$ oc process --local -f policy.json | oc auth reconcile -f -
```

Comment:
The policy file changed to no longer be a "template" type so this full command is now `oc auth reconcile -f policy.json`